pyroSAR/_dev_config.py

```diff
@@ -138,6 +139,23 @@ def __dir__(self):
 # ==============================================================================
 # Class Definitions
 # ==============================================================================
+def make_dir(path=expanduser("~"), dirname='.pyrosar'):
+    directory = os.path.join(path, dirname)
+
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+
+def write_config_file(data, header=None, path=os.path.join(expanduser("~"), '.pyrosar')):
```

Comment:
wouldn't it be better to make header a mandatory input? Otherwise I think your function snap.auxil.get_etc_from_config will throw an IndexError in case somebody has written a line to the file without header. Furthermore, what happens if for some reason a header appears twice. This function should check for exisiting header entries and overwrite them. Sounds much like a class for parsing the config file and setting attributes with simultaneous file write is needed. Feel free to open an issue on this.

Reply:
I will create an issue for this because we should discuss and find a suitable way to solve this.